src/imperative/imperative_utils.h

```diff
@@ -551,17 +539,44 @@ inline bool CheckAndInferStorageType(nnvm::Graph* p_g, const Context& ctx,
   if (node_range.second > node_range.first) {
```

Comment:
The context vector is not needed anymore right? We can just use dev_mask vector

Reply:
Do you mean replace all occurence of `g.attrs.context` with `g.attrs['dev_mask']`??

Reply:
I think context was there for inferstoragetype. Now that inferstoragetype doesn't need context anymore we don't need a context vector

Reply:
Changed to dev_mask now